option.go

```diff
@@ -2,6 +2,7 @@ package gluon
 
 import (
 	"crypto/tls"
+	"github.com/ProtonMail/gluon/imap"
```

Comment:
More weird header ordering 🤔 maybe your editor is using a different autoformatter than mine. Gluon stuff should be below the stdlib stuff.